luarules/gadgets/command_pipeline_handler.lua

```diff
@@ -0,0 +1,48 @@
+local gadget = gadget
+
+function gadget:GetInfo()
+    return {
+        name      = "Reclaim Income Handler",
+        desc      = "Handles reclaim income processing and taxation",
+        author    = "TeamTransfer System",
+        date      = "2024",
+        license   = "GNU GPL, v2 or later",
+        layer     = -1,
+        enabled   = true
+    }
+end
+
+if not gadgetHandler:IsSyncedCode() then
+    return
+end
+
+local ReclaimIncome = VFS.Include("luarules/modules/command_pipeline.lua")
+
```

Comment:
This file. Much like gadgetManager, gives me trepidation because of its limited scope. It does accomplish our ReclaimIncome hook, and maybe let's wait off on expanding it, but maybe give me some hint as to what that would look like if we expanded this system to cover all RecvLuaMsg. Can we not hook an existing mechanic for RecvLuaMsg?